packages/p2p/src/components/my-profile/my-profile-stats/my-profile-stats.scss

```diff
@@ -1,5 +1,9 @@
 .my-profile-stats {
     &-separator {
         margin: 1.6rem 0;
+
+        @include mobile {
+            margin: 1.6rem -1.6rem;
```

Comment:
Increase separator lines to occupy full width, `width:100%` would not work since there is `padding: 1.6rem` applied for responsive screen wrapped in `my-profile__content`:

### Before
![Screenshot 2022-08-03 at 12 46 53 PM](https://user-images.githubusercontent.com/103016120/182529941-3952eea3-5855-4203-8540-b224d155f5be.png)


### After
![Screenshot 2022-08-03 at 12 41 50 PM](https://user-images.githubusercontent.com/103016120/182529963-80c28e05-bbb9-4809-9278-2181127db4fe.png)